docs/src/running-validator/validator-start.md

```diff
@@ -301,6 +302,8 @@ the validator will automatically log to a file.
 The ledger will be placed in the `ledger/` directory by default, use the
 `--ledger` argument to specify a different location.
 
+For validators running on Mainnet-Beta it is strongly recommended to use `--private-rpc` as well as to block the RPC port on the OS firewall or network level (default port is 8899 or the one set explicitly with `--rpc-port` asll we as the websocket port 8900 or the custom RPC port + 1).
```

Comment:
RPC port blocking isn't really needed anymore when `--private-rpc` is provided as the default RPC bind address will be 127.0.0.1, unless `--rpc-bind-address` is also provided